src/dotnet-bootstrapper/Commands/Install/InstallCommand.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine.Parsing;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Deployment.DotNet.Releases;
+
+namespace Microsoft.DotNet.Tools.Bootstrapper.Commands.Install;
+
+internal class InstallCommand(
+    ParseResult parseResult) : CommandBase(parseResult)
+{
+    private string _version = parseResult.ValueForArgument(InstallCommandParser.VersionArgument);
+    private string _rid = BootstrapperUtilities.GetRID();
+    private bool _allowPreviews = parseResult.ValueForOption(InstallCommandParser.AllowPreviewsOption);
+
+
+    public override int Execute()
+    {
+        // If no channel is specified, use the default channel.
+        if (string.IsNullOrEmpty(_version))
+        {
+            _version = BootstrapperUtilities.GetMajorVersionToInstallInDirectory(
+                Environment.CurrentDirectory);
+        }
+
+        ProductCollection productCollection = ProductCollection.GetAsync().Result;
+        Product product = productCollection
+            .FirstOrDefault(p => string.IsNullOrEmpty(_version) || p.ProductVersion.Equals(_version, StringComparison.OrdinalIgnoreCase));
+
+        if (product == null)
+        {
+            Console.WriteLine($"No product found for channel: {_version}");
+            return 1;
+        }
+
+        ProductRelease latestRelease = product.GetReleasesAsync().Result
+            .Where(release => !release.IsPreview || _allowPreviews)
+            .OrderByDescending(release => release.ReleaseDate)
+            .FirstOrDefault();
+
+        if (latestRelease == null)
+        {
+            Console.WriteLine($"No releases found for product: {product.ProductName}");
+            return 1;
+        }
+
+        Console.WriteLine($"Installing {product.ProductName} {latestRelease.Version}...");
+
+        string installationDirectoryPath = BootstrapperUtilities.GetInstallationDirectoryPath();
+
+        foreach (ReleaseComponent component in latestRelease.Components)
+        {
+            Console.WriteLine($"Installing {component.Name}...");
+            DownloadAndExtractReleaseComponentFiles(component, installationDirectoryPath);
+        }
+
+        return 0;
+    }
+
+    private static void DownloadAndExtractReleaseComponentFiles(ReleaseComponent component, string basePath)
+    {
+        if (component is WindowsDesktopReleaseComponent && !OperatingSystem.IsWindows())
+        {
+            return;
+        }
+
+        ReleaseFile releaseFile = component.Files.FirstOrDefault(file =>
```

Comment:
```suggestion
        ReleaseFile releaseArchiveFile = component.Files.FirstOrDefault(file =>
```

nit: naming of the variable could specify that it is a file that represents just the component that has the zip or tar.gz, instead of any part of the release file. If I understand the code correctly 🤔